liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/applications/InstallAppMojoSupport.java

```diff
@@ -68,7 +63,13 @@ protected void installApp(Artifact artifact) throws Exception {
             throw new MojoExecutionException(messages.getString("error.install.app.missing"));
         }
         
-        File destDir = new File(serverDirectory, appsDirectory);
+        setAppsDirectory();
+        
+        if (getAppsDirectory().equalsIgnoreCase("apps") && !isApplicationConfigured()) {
+            addAppConfiguration(artifact.getArtifactId());
```

Comment:
What happen if stripVersion = false and the application filename contains the version string?